src/routes/cart.route.js

```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express';
+import cookieParser from 'cookie-parser';
+import { isAuthenticated } from '../middleware';
+import { clearCartItems } from '../controllers';
```

Comment:
Its better to create a single controller for carts. like `cart.controller.js`

Reply:
no other cart.controler their, you know i am waiting the saidi's work, so I am avoiding conflicts with his work, if the files named the same

Reply:
but let me do it